### ISSUE TITLE
[Documentation] Write a guide on ensuring that colors are accessible

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#3875]
## Definition of done

- [ ] There is a readme in the repo's root `docs/` folder dedicated to accessibility.
- [ ] There is a section in this readme discussing color contrast ratios.
- [ ] The section gives the following recommendations for custom colors on components:
1. When colors are known ahead of time, use https://webaim.org/resources/contrastchecker/
2. When choosing text colors for dynamic background colors, use MDFTypography.

---
#### Internal data
- Associated internal bug: [b/117178433](http://b/117178433)

**Comment by romoore** [comment]
I think this should be for a specific component, @jverkoey can you clarify which component these docs are targeting?

**Comment by jverkoey** [comment]
This is intended to be a general guide on choosing accessible colors and intentionally does not target any specific component, though components may link to this guide where appropriate.

**Comment by romoore** [comment]
Ok, then I don't think this is strictly required for our accessibility levels project.

**Comment by romoore** [comment]
I still don't think this is required for the accessibility levels project.

**Comment by material-mirror** [comment]
The internal issue [b/117178433](http://b/117178433) is now closed. This issue is being closed as a result.
